liquibase-core/src/main/java/liquibase/diff/output/changelog/core/MissingDataExternalFileChangeGenerator.java

```diff
@@ -139,7 +139,7 @@ public Change[] fixMissing(DatabaseObject missingObject, DiffOutputControl outpu
                 }
 
                 LoadDataChange change = new LoadDataChange();
-                change.setFile(fileName);
+                change.setFile(externalFileResource.getPath());
```

Comment:
What gets stored in here? Is it a searchpath relative path like `mydata/populated.csv` ? 

Or does it have a "direct" path like `s3:/bucket/mydata/populated.csv` or `/my-project/src/mydata/populated.csv`  ?

Reply:
Relative, like your first example: `mydata/populated.csv` if your `--data-output-directory` were `mydata`

Reply:
OK, sounds good. We don't want it to be absolute.